packages/smooth_app/lib/pages/product/common/product_query_page.dart

```diff
@@ -196,15 +202,23 @@ class _ProductQueryPageState extends State<ProductQueryPage>
             ),
           ],
         ),
-        appBar: AppBar(
+        appBar: SmoothAppBar(
           backgroundColor: themeData.scaffoldBackgroundColor,
           elevation: 2,
           automaticallyImplyLeading: false,
           leading: const SmoothBackButton(),
           title: _AppBarTitle(
-            name: widget.name,
+            name: widget.searchResult
+                ? widget.name
+                : appLocalizations.product_search_same_category,
             editableAppBarTitle: widget.editableAppBarTitle,
           ),
+          subTitle: !widget.searchResult
+              ? _AppBarTitle(
```

Comment:
The goal of an AppBarTitle is to support both title and subtitles.
You don't use it the right way here

Reply:
I'm not sure what that means, Would be nice if you could give a little explanation about what you want

Reply:
An `AppBarTitle` does have a title and a subtitle.
So instead, just pass an `AppBarTitle` to the title attribute and pass the title to the `AppBarTitle` object

Reply:
There are 2 versions of the `_AppBarTitle` one in this file that takes `name` and `editableAppBarTitle` as arguments, one from the `smooth_app_bar.dart` the one that you mention.
Do you want me to replace the widget in this file with the one with the `title` and the `subtitle`?

Reply:
That's the same AppBarTitle you use two lines above